.mypy.ini

```diff
@@ -38,24 +38,10 @@ ignore_missing_imports = True
 [mypy-ert._clib.*]
 ignore_missing_imports = True
 
-[mypy-ert.gui.model.node]
-ignore_missing_imports = True
-ignore_errors = True
-
-[mypy-ert.gui.model.snapshot]
-ignore_missing_imports = True
-ignore_errors = True
-
 [mypy-ert.gui.ertwidgets.validationsupport]
-ignore_missing_imports = True
 ignore_errors = True
```

Comment:
Why do we want to keep this one?

Reply:
because it still has typing issues :melting_face: